README.md

```diff
@@ -2,7 +2,94 @@
+Deprecated versions will be tagged as a specific version, eg. `v0.1` and may be supported indefinitely.
+
+Deprecated versions as of this commit:
+
+- `v0.1`: RRP-based Beacons
+
+New deployments must follow the `new branch->PR (including CI status checks)->merge to main` process.
+
+## Chain and contract deployment references
+
+API3 operational services rely on various on-chain contracts. Deployments of these contracts are referenced by their
+chain name in the
+[API3 utility-contracts repository](https://github.com/api3dao/utility-contracts/tree/main/SelfServeRrpBeaconServerWhitelister/deployments).
+Deployed contracts are versioned through their import in `package.json`, which can be any supported git reference.
+
+---
+
+## Integrating a new API
+
+This repository contains utilities for generating and integrating new APIs. Utilities rely on `yarn` and various
+dependencies and therefore these should be installed before utilising any utilities with the command `yarn install`.
+
+Utilities take the form of interactive CLI applications. The following utilities, in deployment order, are available:
+
+- `create-boilerplate`: Prompts for various API metadata and creates a set of folders with examples. These folders must
+  be populated with real data and the example files removed prior to running the next step.
+  - The files that must be supplied and/or edited will be:
+    - `templates/*`
+    - `beacons/*`
+    - `ois/*`
+      - Note that the above files can have any name initially as the validator will name them according to their
+        `templateName`, `beaconName` and `oisTitle-oisVersion`.
+    - `deployments/secrets.env`:
+      - This is a boilerplate file, but it contains a programmatically generated HTTP gateway endpoint secret. Please
+        retain this secret and post it to our `#operations-keys` Slack channel on successful deployment, along with
+        `receipt.json`.
+      - This secret should not be committed and will be expunged pre-commit.
+    - Please reference the "Directory and file structure" section below for more information on these resources.
+- `validate`: Validates and conforms the entire repository - any errors must be fixed prior to continuing.
+- `create-config`: Creates a set of config files:
+  - `config.json`: The Airnode config file, should not require editing.
+  - `airkeeper.json`: The Airkeeper config file, should not require editing.
+  - `secrets.env`: This file will need to be populated prior to executing the next step and ultimately deployment.
+    - The file will require an HTTP gateway secret, this secret will be generated and populated in the next deployment
+      step on the API provider's machine.
+    - Important: Once the deployment is complete, the HTTP Gateway key, along with the `receipt.json` file must be
+      posted to the `#operations-keys` channel on our Slack workspace.
+  - The above files will be validated by their respective validators. Be sure to fix any errors output.
```

Comment:
We have a separate validator only for Airnode. Airkeeper and Airseeker will at most have a zod schema and it's currently not built to be used from outside (afaik). So, keep that in mind and let me know so I can create tasks for that if need be.

Reply:
We will need to validate the operations repository as it'll be our single source of truth, so please go ahead and create tasks to expose the existing validators :)